IDR/notebook/scripts/nbconverted/1.describe_screens.py

```diff
@@ -267,51 +390,46 @@ def describe_screen(screen_id):
 screen_details_df.to_csv(output_file, index=False, sep="\t")
 
 
-# In[13]:
+# In the chunk below, we collect and process a subset of the available IDR screens. The subset contains IDR accession names idr0080, idr0001, and idr0069. The idr0080 dataset was collected by Way et al. (2021) and will be used as a validation set for our statistics pipeline as we already know the image counts and unique attribute counts. The other two datasets are used for prototyping iterative functionalities of metadata collection and statistics.
+
+# In[40]:
+
+
+# Collect study_names and imaging method metadata for each screen
+idr_names_dict = dict()
+for index in screen_details_df.itertuples(index=False):
+    screenID = index[19]
+    idr_name = index[18]
+    img_type = index[5]
+    sample = index[0]
+    idr_names_dict[idr_name] = [screenID, img_type, sample]
+
+# Subset of studies for prototyping
+idr_meta_dict = dict(
```

Comment:
i think `dict(zip())` might be the function you're looking for here. It feels more pythonic to me. (might be worth looking into)

https://www.programiz.com/python-programming/methods/built-in/zip

Reply:
@gwaybio I didn't need the dictionary at all and was doing a few unnecessary steps. Instead I used `list(zip())` to generate a list of tuples from subsets of the data from the main dictionary. I didn't know about this functionality before so thanks!

Added in 364ed2ccf8fca7bf3978fcc9124ad4c9b2ee878a.